web_src/js/features/repo-editor.js

```diff
@@ -7,9 +7,9 @@ import {attachRefIssueContextPopup} from './contextpopup.js';
 import {POST} from '../modules/fetch.js';
 
 function initEditPreviewTab($form) {
-  const $tabMenu = $form.find('.tabular.menu');
+  const $tabMenu = $form.find('.repo-editor-menu');
   $tabMenu.find('.item').tab();
-  const $previewTab = $tabMenu.find(`.item[data-tab="${$tabMenu.data('preview')}"]`);
+  const $previewTab = $tabMenu.find('a[data-tab="preview"]');
```

Comment:
Why require it to be a link?
```suggestion
  const $previewTab = $tabMenu.find('[data-tab="preview"]');
```

Reply:
Because the `data-tab` attribute is used for both the link to the tab and the tab content element itself, so we can not query on data attribute alone. Doing this change would most likely break the tabs.